CMakeLists.txt

```diff
@@ -163,6 +163,7 @@ cmake_dependent_option(AIECC_LINK_WITH_XCHESSCC
 
 
 add_flag_if_supported("-Werror=sign-compare" WERROR_SIGN_COMPARE)
+add_flag_if_supported("-Werror=switch-enum" WERROR_SWITCH_ENUM)
```

Comment:
That's much better!

Reply:
It doesn't work - upstream has a failing case https://github.com/llvm/llvm-project/blob/main/llvm/include/llvm/ADT/Twine.h#L443-L451. I plan to submit a patch but who knows what they'll say.